src/pages/Login/Component/MainForm/MainForm.js

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import Input from '../../../../Component/Input/Input';
+import InputWrap from '../../../../Component/InputWrap/InputWrap';
+import Button from '../../../../Component/Button/Button';
+import IconButton from '../../../../Component/IconButton/IconButton';
+import './MainForm.scss';
+
+const MainForm = props => {
+  const { onChange } = props;
+  const navigate = useNavigate();
+  const [userInfo, setUserInfo] = useState({
+    userId: '',
+    userPw: '',
+  });
+
+  const { userId, userPw } = userInfo;
+  const onChangeCheck = e => {
+    const { name, value } = e.target;
+
+    setUserInfo({ ...userInfo, [name]: value });
+  };
+
+  const onSubmit = e => {
+    e.preventDefault();
+  };
+
+  const loginValidation = e => {
+    const idRegExp = /^[A-Za-z0-9]{6,}$/;
+    const passwordRegEx =
+      /^(?=.*[a-zA-Z])(?=.*[!@#$%^*+=-])(?=.*[0-9]).{8,20}$/;
+
+    if (!idRegExp.test(userId)) {
+      // alert('6글자 이상 소문자와 숫자로 혼합된 아이디를 입력해주세요.');
+      return false;
+    } else if (!passwordRegEx.test(userPw)) {
+      // alert(
+      //   '비밀번호는 영문, 숫자, 특수기호를 조합하여 8~20글자 이상 입력해주세요.',
+      // );
+      return false;
+    } else {
+      return true;
+    }
+  };
```

Comment:
alert를 출력하지 않는다면, 이렇게 변수로 사용하시면 될 것 같습니다.
```suggestion
  const idRegExp = /^[A-Za-z0-9]{6,}$/;
  const passwordRegEx = /^(?=.*[a-zA-Z])(?=.*[!@#$%^*+=-])(?=.*[0-9]).{8,20}$/;
  const isLoginInputValid = idRegExp.test(userId) && passwordRegEx.test(userPw);
```